app/frontend/Calendar/EditEvent/DialogHeader.svelte

```diff
@@ -235,19 +235,12 @@
   }
 
   async function onChangeRemainder() {
-    let master = event.calendar.newEvent();
-    master.startEditing();
-    master.copyEditableFieldsFrom(event);
-    master.calUID = null;
-    if (event.parentEvent.recurrenceRule) {
-      let { frequency, interval, week, weekdays } = event.parentEvent.recurrenceRule;
-      master.newRecurrenceRule(frequency, interval, week, weekdays);
-    }
-    await saveEvent(master);
-    master.finishEditing();
+    let newMaster = event.cloneSeriesFromThisInstance();
+    await newMaster.save();
```

Comment:
`saveEvent` used to add the event to the calendar.
```suggestion
    let newMaster = event.cloneSeriesFromThisInstance();
    await newMaster.save();
    newMaster.calendar.events.add(newMaster);
```